CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,65 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Validate if Deferred Account is set for Financial Type
+   * when Deferred Revenue is enabled
+   *
+   * @return string
+   *
+   */
+  public static function validateTogglingDeferredRevenue() {
+    $deferredFinancialType = self::getDeferredFinancialType();
```

Comment:
my understanding is that you need to do the translate at this point for the translators to be able to define the strings